src/wallet/wallet.cpp

```diff
@@ -3293,6 +3296,12 @@ std::string CWallet::GetWalletHelpString(bool showDebug)
 
 bool CWallet::InitLoadWallet()
 {
+    if (GetBoolArg("-disablewallet", false)) {
```

Comment:
But for consistency it'd make sense to have one. When changing the default it's easy enough to forget one of the GetArgs().